docs/schema/module-element-site.md

```diff
@@ -39,16 +39,16 @@ The following sections describe attributes, child elements, and parent elements.
 
 ### Attributes
 
-|**Attribute**|**Description**|
+| Attribute | Description |
```

Comment:
Let's add the asterisks back to this table.  I'm not sure if this table needed any changes.